week2/homework/src/index.js

```diff
@@ -1,3 +1,118 @@
 'use strict';
+let fs = require('fs');
```

Comment:
Please consider using `const` over `let` as much as possible.
It helps defending you against bugs and prevents you changing stuff that shouldn't be changed.
You can [read more here](https://wesbos.com/let-vs-const/)

Reply:
yes i have read the topic and using const can be more logical. we do not need any update on those variables. Corrected